sdk/devcenter/azure-developer-devcenter/tests/recordings/test_devcenter_operations.pyTestDevcentertest_devcenter_operations.json

```diff
@@ -0,0 +1,215 @@
+          "preferred_username",
+          "name",
+          "tid",
+          "ver",
+          "at_hash",
+          "c_hash",
+          "email"
+        ],
+        "kerberos_endpoint": "https://login.microsoftonline.com/organizations/kerberos",
+        "tenant_region_scope": null,
+        "cloud_instance_name": "microsoftonline.com",
+        "cloud_graph_host_name": "graph.windows.net",
+        "msgraph_host": "graph.microsoft.com",
+        "rbac_url": "https://pas.windows.net"
+      }
+    },
+    {
+      "RequestUri": "https://login.microsoftonline.com/organizations/oauth2/v2.0/token",
+      "RequestMethod": "POST",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip, deflate",
+        "client-request-id": "ef5ea3de-386c-4504-807a-ba7437458233",
+        "Connection": "keep-alive",
+        "Content-Length": "1344",
+        "Content-Type": "application/x-www-form-urlencoded",
+        "Cookie": "fpc=AvPprpzQdVxOjEDVBZf5zAA; stsservicecookie=estsfd; x-ms-gateway-slice=estsfd",
+        "User-Agent": "azsdk-python-identity/1.13.0 Python/3.10.5 (Windows-10-10.0.22621-SP0)",
+        "X-AnchorMailbox": "Oid:df428e89-1bc2-4e72-b736-032c31a6cd97@8ab2df1c-ed88-4946-a8a9-e1bbb3e4d1fd",
+        "x-client-cpu": "x64",
+        "x-client-current-telemetry": "4|169,0|",
+        "x-client-last-telemetry": "4|0|||",
+        "x-client-os": "win32",
+        "x-client-sku": "MSAL.Python",
+        "x-client-ver": "1.22.0",
+        "x-ms-lib-capability": "retry-after, h429"
+      },
+      "RequestBody": "client_id=04b07795-8ddb-461a-bbee-02f9e1bf7b46\u0026grant_type=authorization_code\u0026client_info=1\u0026claims=%7B%22access_token%22%3A\u002B%7B%22xms_cc%22%3A\u002B%7B%22values%22%3A\u002B%5B%22CP1%22%5D%7D%7D%7D\u0026code_verifier=U87KAR4ZLcBvadoVQmkD5x9W.y0G~bjweCTihFEYSJM\u0026code=0.AQEAHN-yiojtRkmoqeG7s-TR_ZV3sATbjRpGu-4C-eG_e0YaANg.AgABAAIAAAD--DLA3VO7QrddgJg7WevrAgDs_wUA9P_AbRGB7ffQ99luaD2_9Z31i2PN9dCXvTyi7SiG5y3VDBl91gGj4R7kR9OhbInyuvzRoJwzNPVO13Mi0sBw5ywngfpslsBrxQK3LngSpWpiwKRpckZjJ_TpdxvSg_NHEanAt2jBamdyn1dw5KGSCUWiZZ5DxmwuJxelXT87nF-Ww8M8xkn87IWPo-Yz0NqZU--QyC2AGX2b0ENOf8Fpl92tMZWxEBkswRByUYVmOqesWSXN5Kqdv-6fU7quwbqVyyAskukvMuDmkxx_YR9DMAyjfcvIvdV7PywN2212ecOyXi29G3kU6Phd8mIb1usJ137ojx0phsQD761UhShkDp9nzpTcXpeYVfSLUBAjBMlHqD_rP_Lqr2ytwGL-dj5mkDBNGgCeAKl7radsLKDgA1AEaQUL1fobd8NkCcXkKRboiMEcfogF5uQ_Owh1ePOP2FsnC70Zgrx766PecmcV1viG6c7FELHCQR7LPxYWbZGjOcOw7C6zNHTnFO6wqOpdZSMazvwe1vVq_t256zNrBN10AK-8O_L50edgs_7rTEWOkrSlh-qerudx0e-qEcsflnQV1U8IJmv2YcNSRKozVkMXMWHbuLaMZRV70_WkjW1wExMYRkdq0Pfxoa1m2aPtdTO5HBS1nJfcLMDBJZSavt98VYinqyLyBVLeHgazkRhVxS1vO8TCoSsTLpdgGafoRceGkcV11IexzvxuxxsERo_8S_ViGg1Tlh4JO4Clb5LHwnqu16R4udaCWa_S8O7Zxn-06xLysxyANRp5yFfCHZWr0Vn7TgNqYsmOFAwiLD17RCGb_1uuy30khm_h41X-sprkDurSXr92NcqCZZSJnngZyqmxCSAl9tTPzmtwHeXD\u0026redirect_uri=http%3A%2F%2Flocalhost%3A63536\u0026scope=https%3A%2F%2Fdevcenter.azure.com%2F.default\u002Boffline_access\u002Bopenid\u002Bprofile",
```

Comment:
This is odd. For whatever reason the body didn't seem to be processed correctly by the test proxy, so regular sanitization didn't occur.

To prevent issues like this, and assuming authentication requests aren't important for your recordings (which I wouldn't expect them to be), I recommend adding a call to [`add_oauth_response_sanitizer()`](https://github.com/Azure/azure-sdk-for-python/blob/497f5f3435162c4f2086d1429fc1bba4f31a4354/tools/azure-sdk-tools/devtools_testutils/sanitizers.py#L334) in your `conftest.py` setup. This will remove any `/oauth2/v2.0/` interactions, like this request

Reply:
I've added the oauth sanitizer, but this "odd request looking" will just be updated for the upcoming recordings